spec/features/front_end_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "spec_helper"
+
+RSpec.describe "Front end" do
+  before :all do
+    drop_dummy_database
+    remove_project_directory
+  end
+
+  let(:gemfile) { IO.read("#{project_path}/Gemfile") }
+  let(:node_modules_file) { IO.read("#{project_path}/package.json") }
+  let(:application_js_file) { IO.read("#{project_path}/app/javascript/packs/application.js") }
+  let(:layout_file) { IO.read("#{project_path}/app/views/layouts/application.html.erb") }
+  let(:application_css_file) { IO.read("#{project_path}/app/javascript/css/application.css") }
+  let(:tailwind_config_file) { IO.read("#{project_path}/tailwind.config.js") }
+  let(:rails_css_file) { IO.read("#{project_path}/app/assets/stylesheets/application.css") }
+
+  it "creates a project wihtout a front end framework" do
+    remove_project_directory
+    create_dummy_project("front_end" => "None")
+    expect(gemfile).to include('webpacker')
+  end
+
+  context "with vue" do
+    before(:all) do
```

Comment:
Beware of using before(:all) as it may cause state to leak between tests. If you are using rspec-rails, and use_transactional_fixtures is enabled, then records created in before(:all) are not automatically rolled back.